Match text and JSON output

hmdros opened the pull request:
Closes #39 

- [ ] Add an entry to `CHANGELOG.md` that links to this PR under the "main (unreleased)" heading.

**Description:**

Add "Code" and "Tests" total count from bottom of report to JSON output

I will abide by the [code of conduct](CODE_OF_CONDUCT.md).

etagwerker reviewed (approved):
@hmdros Looks good! Could you please add an entry to the CHANGELOG file?

hmdros commented:
> @hmdros Looks good! Could you please add an entry to the CHANGELOG file?

Hey @etagwerker I added the CHANGELOG entry. 
Later I noticed the gem were missing a call to `Bundler::Stats::CLI.start(["-f", "json"])` to match the initial block from the console output and added it now. Do you mind reviewing it again?

hmdros commented on README.md:
I did reduce the number of examples to 1. Also the output is a little bigger now, since we bring more information
